dotcom-rendering/src/web/components/Article.tsx

```diff
@@ -25,7 +25,7 @@ type Props = {
 
 /**
  * @description
- * Article is a high level wrapper for pages on Dotcom. Sets strict mode and some globals
+ * Article is a high level wrapper for article pages on Dotcom. Sets strict mode and some globals
```

Comment:
For consistency I would rename this file.

```suggestion
 * ArticlePage is a high level wrapper for article pages on Dotcom. Sets strict mode and some globals
```

Reply:
Yeah, this makes a lot of sense. Implemented